packages/charts/src/chart_types/heatmap/state/selectors/on_brush_end_caller.ts

```diff
@@ -27,7 +27,7 @@ import { DragCheckProps, hasDragged } from '../../../../utils/events';
 export function createOnBrushEndCaller(): (state: GlobalChartState) => void {
   let prevProps: DragCheckProps | null = null;
   let selector: OutputSelector<
-    GlobalChartState,
+    [(state: GlobalChartState) => GlobalChartState],
```

Comment:
This should be an array of `Selectors` not a tuple of one `Selector`.

```suggestion
    Array<(state: GlobalChartState) => GlobalChartState>,
```

Reply:
Nice catch, fixed it in [0a2258e](https://github.com/elastic/elastic-charts/pull/2607/commits/0a2258e3d9a8151f3b41e4aca83b6892acf6d622).